msteams-platform/apps-in-teams-meetings/build-apps-for-teams-meeting-stage.md

```diff
@@ -266,19 +266,125 @@ You can build an in-meeting app for enabling meeting participants to sign docume
 
 :::image type="content" source="../assets/images/sbs-inmeeting-doc-signing/final-output.png" alt-text="In-meeting document signing app":::
 
 The following participant roles may be involved during the meeting:
 
-- **Document creator**: This role can add their own documents to be reviewed and signed.
-- **Signer**: This role can sign reviewed documents.
-- **Reader**: This role can view the documents added to the meeting.
+* **Document creator**: This role can add their own documents to be reviewed and signed.
+* **Signer**: This role can sign reviewed documents.
+* **Reader**: This role can view the documents added to the meeting.
+
+## App caching
+
+App caching helps you to improve launch time of the apps that are loaded in the meeting side panel.
+
+> [!NOTE]
+>
+> * Currently app caching is available only in [public developer preview](~/resources/dev-preview/developer-preview-intro.md).
+> * App caching is supported only for tabs loaded in the meeting side panel in Teams desktop client.
+> * App caching is not supported on chats, channels, and personal apps.
+
+### Enable app caching
+
+To enable app caching in your meeting side panel, follow the steps:
+
+1. Call `microsoftTeams.registerBeforeUnloadHandler` and `microsoftTeams.registerOnLoadHandler`.
+
+1. Use `contentUrl` and `entityId` into the load handler to route to the correct page within your app and invoke `notifySuccess/notifyFailure` to notify Teams client that the app initialization flow is complete.
+
+1. Dispose resources and perform any cleanup needed in the `beforeUnload handler`, then invoke the `readyToUnload` callback to notify Teams client that the app unload flow is complete.
+
+The following is the flow diagram of the app added to the meeting stage without app caching:
+
+:::image type="content" source="../assets/images/saas-offer/first-launch-app.png" alt-text="This screenshot shows the flow of the first launch of the app in meeting stage.":::
+
+The following is the flow diagram of a cached app when it's added to the meeting stage:
+
+:::image type="content" source="../assets/images/saas-offer/cached-launch-app.png" alt-text="This screenshot shows the flow of the cached launch of the app in meeting stage.":::
+
+App caching hosts the embedded app in webview and reused when users go to different instances of the app within a window.
+
+The webview of the app remains in the DOM. The webview is hidden when the users go out of the app and shown when the users return to the app. When the app is cached, any audio that is playing is muted.
+
+> [!NOTE]
+> If the app caching is not enabled, the webview is recreated every time the users go out and return to the app.
+
+Following are the parameters to control the conditions for the apps to be added or removed from the cache:
+
+* The maximum cache size for apps in meetings is 1 MB. When the cache size is exceeded, the app is removed from the cache.
```

Comment:
Just 1. this is the max number of Apps in the Cache. When the cache size is exceeded, the least recently used App will be evicted